tests/components/kodi/test_init.py

```diff
@@ -0,0 +1,25 @@
+"""Test the Kodi integration init."""
+from homeassistant.components.kodi.const import DOMAIN
+from homeassistant.config_entries import ENTRY_STATE_LOADED, ENTRY_STATE_NOT_LOADED
+
+from . import init_integration
+
+from tests.async_mock import patch
+
+
+async def test_unload_entry(hass):
+    """Test successful unload of entry."""
+    with patch(
+        "homeassistant.components.kodi.media_player.async_setup_entry",
+        return_value=True,
+    ):
+        entry = await init_integration(hass)
+
+    assert len(hass.config_entries.async_entries(DOMAIN)) == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6042836)